client/main.lua

```diff
@@ -47,9 +47,7 @@ end
 local function updatePlantStats()
     if not (insideHouse or plantsSpawned) then return end
     for k, v in pairs(housePlants[currentHouse]) do
-        local gender = 'M'
-        if v.gender == 'woman' then gender = 'F' end
-
+        local gender = (v.gender == 'female') and 'F' or 'M'
```

Comment:
```suggestion
        local gender = v.gender == 'female' and 'F' or 'M'
```

We don't need the brackets here